src/NCronJob/Registry/JobDefinition.cs

```diff
@@ -47,4 +47,31 @@ internal sealed record JobDefinition(
     private JobExecutionAttributes JobPolicyMetadata { get; } = JobPolicyMetadata ?? new JobExecutionAttributes(Type);
     public RetryPolicyBaseAttribute? RetryPolicy => JobPolicyMetadata.RetryPolicy;
     public SupportsConcurrencyAttribute? ConcurrencyPolicy => JobPolicyMetadata.ConcurrencyPolicy;
+
+    private bool IsDisabled => CronExpression == TheThirtyFirstOfFebruary;
+
+    public bool IsEnabled => CronExpression is null || !IsDisabled;
+
+    public void Disable()
+    {
+        // Scheduling a job on Feb, 31st is a sure way to never get it to run
+        CronExpression = TheThirtyFirstOfFebruary;
+    }
+
+    public void Enable()
+    {
+        if (UserDefinedCronExpression is not null)
+        {
+            CronExpression = CronExpression.Parse(UserDefinedCronExpression);
+            return;
+        }
+
+        CronExpression = null;
+    }
+
+    public DateTimeOffset? GetNextCronOccurrence(DateTimeOffset utcNow, TimeZoneInfo? timeZone)
+        => CronExpression?.GetNextOccurrence(utcNow, timeZone);
+
+    // https://crontab.guru/#*_*_31_2_*
+    private static readonly CronExpression TheThirtyFirstOfFebruary = CronExpression.Parse("* * 31 2 *");
```

Comment:
I would still call it something like `DisabledCronJob` or `NotReacheableCronDefinition` or so. That would remove the comment as well

Reply:
I'll take care of that in a follow-up PR.